src/ExpressionEvaluator/CSharp/Source/ResultProvider/NetFX20/project.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "Microsoft.NetFX20": "1.0.3",
-    "Microsoft.VisualStudio.Debugger.Engine": "14.3.25420",
-    "Microsoft.VisualStudio.Debugger.Metadata": "14.3.25420"
+    "Microsoft.VisualStudio.Debugger.Engine": "14.3.25421",
+    "Microsoft.VisualStudio.Debugger.Metadata": "14.3.25421"
```

Comment:
The core ResultProvider project has the references right? Why not just remove these and let it flow?